sql/analyzer/validate_create_table.go

```diff
@@ -90,11 +94,18 @@ func validateAlterColumn(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope
 			if err != nil {
 				return nil, err
 			}
+		case *plan.DropColumn:
+			sch, err = validateDropColumn(initialSch, sch, n)
+			if err != nil {
+				return nil, err
+			}
 		}
 		return n, nil
 	})
 }
 
+// Schema is passed in twice, because one version is the initial version before the alter column expressions are
```

Comment:
Start golang function comments with the name of the function and a one-sentence description of what it does

Reply:
✅